### ISSUE TITLE
Can't create record (maps2 4.0.1/TYPO3 8.7.16)

### ISSUE THREAD

**Issue by wini2** [jweiland-net/maps2#61]
Creating a record fails because i can't fill 'Adresse'-Field (Bitte verwenden Sie das Suchfeld in der Google-Karte, um dieses Feld zu beüllen). If I use the searchfield on the map nothing happens.

**Comment by froemken** [comment]
Yes. Wolfgang told me about it, too. You need to activate Google Maps AND Google Places in Google Console. I have to add this information to documentation. Sorry.
